ironfish-cli/src/commands/miners/start.ts

```diff
@@ -33,6 +33,12 @@ export class Miner extends IronfishCommand {
       char: 'p',
       description: 'the host and port of the mining pool to connect to such as 92.191.17.232',
     }),
+    name: Flags.string({
+      char: 'n',
+      default: 'default',
+      description:
+        'the miner name distinguish different machines such as iron1. Server hostname is recommended',
```

Comment:
we don't need to make a recommendation, let the user do what they want.

```suggestion
        'the miner name distinguishes different miners',
```